src/tex/appendix.tex

```diff
@@ -447,6 +447,18 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone model consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.47 on \ChemBench corpus, which is only 18\% less accurate than the larger \LlamaThreeOneEightBInstruct model.
+One can observe a clear progression of performance across model families (e.g., \ClaudeThreeFiveSonnet >  \ClaudeThree > \ClaudeTwo). Newer versions consistently outperform their predecessors, indicating improvements in scientific understanding of these model families.
+Scores of knowledge-intensive are generally lower than those of Calculation and Reasoning. Reasoning tends to be the strongest skill among the three.
+
+Based on these findings, some scope for improving these models could be to focus on enhancing knowledge-based training, possibly through improved pre-training on chemistry-specific texts or with chemistry knowledge-base integration. 
+Models show the strongest performance in reasoning tasks, which suggests that current training approaches are good at developing logical/reasoning capabilities. Larger models consistently perform better. However, there's evidence that well-trained smaller models can compete with larger models. 
+Performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions should be included in training or finetuning.
```

Comment:
```suggestion
Models show the strongest performance in reasoning tasks on the \chembench corpus, which suggests that current training approaches are good at developing logical/reasoning capabilities for basic problem-solving of university-level exams. 
However, performance drops significantly for advanced tasks, suggesting that more advanced chemistry problems with complex, multi-step solutions should be included in training or finetuning.
```

perhaps give concrete examples of questions that are still difficult (NMR, isomers, ion lottery) 


in the model scaling discussion, we can perhaps also emphasize that some of the tasks (isomers, point group) show very clear distinctions between models 
<img width="935" alt="Screenshot 2024-10-25 at 16 33 07" src="https://github.com/user-attachments/assets/4af21a1a-d074-4b11-81ca-7eec77a14ac8">